packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -75,19 +81,31 @@ export class Context implements FlexTreeContext, IDisposable {
 	private disposed = false;
 
 	/**
-	 * @param forest - the Forest
-	 * @param editor - an editor that makes changes to the forest.
+	 * The forest from the {@link checkout}.
+	 */
+	public readonly forest: IForestSubscription;
+
+	/**
+	 * An editor that makes changes to the {@link forest}.
+	 */
+	public readonly editor: IDefaultEditBuilder;
```

Comment:
Decided to keep these here even though they're accessible through the checkout for simplicity, but also because the editor is typed as `IDefaultEditBuilder`, whereas `ITreeCheckout.editor` is a `ISharedTreeEditor` which also allows schema changes. Not sure if we would want/need to expose that here.

Whether we keep this editor or not, I guess we could remove `forest` and have it be accessed through the context's checkout?